frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/helpers/CommonTestVariables.java

```diff
@@ -20,6 +20,7 @@ public class CommonTestVariables {
             "YTtKVSlub1YlOSBTeEI4J3pVLVd7Jjl8VkBfREs2N3clZmN+fnU7fXNbcTJjKyEzN2IuUXIgMGttV058fGhUZ0xhenZUdldEblB8SH18XypwXUhWPXhYXTNQeURW%";
     public static final String SESSION_ID = "session-id";
     public static final String CLIENT_SESSION_ID = "known-client-session-id";
+    public static final String INTERNAL_SUBJECT_ID = "some-subject-id";
```

Comment:
Might be worth using this in other places that want an internal subject id? E.g., in `MfaResetIPVAuthorizationServiceTest`

Reply:
Let's be really careful not to mix up `internal subject ID` and `internal common subject ID` aka `internal pairwise subject ID`. To help with that, let's make this realistic test data.

Reply:
Think I've made the name clearer and tried to make the subject identifier look a little more realistic